### ISSUE TITLE
Plz i need medivia items ID

### ISSUE THREAD

**Issue by nephtyskiki** [Ascerr/Lua-Scripts#34]
> any help me

**Comment by Ascerr** [comment]
You can only look on Item id if it's visible on screen or inside
container/equipment and ID will be shown in Rifbot Information box.

czw., 25 lip 2024 o 15:16 nephtyskiki ***@***.***> napisał(a):

**Comment by nephtyskiki** [comment]
Yes I know, but I would like to know if I can get a list of the Medivia IDs

**Comment by Ascerr** [comment]
nope

czw., 25 lip 2024 o 21:57 nephtyskiki ***@***.***> napisał(a):
